Makefile

```diff
@@ -6,16 +6,17 @@ GOTOOLS = github.com/golang/lint/golint \
 # Temporary patch to avoid build failing because of the outdated documentation example
 # TODO: uncomment below commented lines and remove any line that uses $(NODOCS)
 NODOCS = $(shell go list ./... | grep -v /docs/)
+PKGS = $(shell go list ./... | egrep -v "\/docs\/|jmx")
 
-all: validate test
+all: lint test
```

Comment:
Do we all agree to name this target as `lint` rather than `validate`? 
If we do, I vote for changing all the rest of Makefile in our code base.

FYI, my opinion here is +1.

Reply:
I don't really mind about the name, we removed the `validate` as it was only a proxy step adding noise